### PR TITLE
Add bulk a deleting multple selected cases

### PR DESCRIPTION
**Pull request by RMadjev**

### CODE REVIEW

File: tcms/testplans/static/testplans/js/get.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-var expandedTestCaseIds = [];
+var expandedTestCaseIds = [],
+    fadeAnimationTime = 500;
```
Comment (atodorov) [inline tcms/testplans/static/testplans/js/get.js:2]: this seems to be old, shouldn't this PR be using the animate function ?

**Review by atodorov — CHANGES_REQUESTED** [review]
The functionality works but looks like it needs to be rebased.

File: tcms/testplans/static/testplans/js/get.js (hunk 1 above)
Comment (RMadjev) [reply tcms/testplans/static/testplans/js/get.js:2]: Yeah, but I noticed that with removing with introduce a bug, when we remove a testcase from a plan we fadeout the row and then remove it from the dom, no fadein is used(animate also will not do the trick).

File: tcms/testplans/static/testplans/js/get.js [hunk 2]
```diff
@@ -358,6 +359,26 @@ function toolbarEvents() {
         }
 
     });
+
+    $('#delete_button').click(function(ev) {
+        let selectedCases = getSelectedTestCases();
+
+        if (!selectedCases.length) {
```
Comment (atodorov) [inline tcms/testplans/static/testplans/js/get.js:366]: Should ask the `Are you sure` question, see the bulk-delete button in refactor_testrun_ui branch.

**Review by atodorov — CHANGES_REQUESTED** [review]

**Review by atodorov — APPROVED** [review]

**Pull request merged**